resources/definitions/fdmprinter.def.json

```diff
@@ -3147,20 +3368,69 @@
                     "settable_per_extruder": true,
                     "children":
                     {
-                        "support_interface_line_distance":
+                        "support_bottom_density":
                         {
-                            "label": "Support Interface Line Distance",
-                            "description": "Distance between the printed support interface lines. This setting is calculated by the Support Interface Density, but can be adjusted separately.",
-                            "unit": "mm",
+                            "label": "Support Bottom Density",
+                            "description": "Adjusts the density of the bottoms of the support structure. A higher value results in better adhesion of the support on top of the model.",
```

Comment:
You switched to a verb form of describing settings.
We usually describe what the setting value means - not what tweaking the setting value does.
Please remove 'Adjusts the'.